### PR TITLE
Dit 2828

### PR DESCRIPTION
**Pull request by ansahdavid**
Updated functionality of the Back link displayed in web pages

### CODE REVIEW

File: test/unit/views/partials/BackLinkViewSpec.scala [hunk 1]
```diff
@@ -55,128 +46,74 @@ class BackLinkViewSpec extends ViewSpec {
...
+      val doc = view(back_link()(messages))
 
       // Then
       doc                             should containElementWithID("back-link")
       doc.getElementById("back-link") should haveTag("a")
-      doc.getElementById("back-link") should haveAttribute("href", "url")
-      doc.text()                      shouldBe "Back to " + customLabel
+      doc.getElementById("back-link") should haveAttribute("href", "#")
+      doc.text()                      shouldBe "Back"
     }
```
Comment (DavidGregory084) [inline test/unit/views/partials/BackLinkViewSpec.scala:117]: I think all of these the tests in this spec are pretty much the same now, so perhaps we could get rid of all but one?

**Review by DavidGregory084 — APPROVED** [review]
Looks good 👍

File: test/unit/views/partials/BackLinkViewSpec.scala (hunk 1 above)
Comment (ansahdavid) [reply test/unit/views/partials/BackLinkViewSpec.scala:117]: Yep good point David, I'll update them now.

Comment (ansahdavid) [reply test/unit/views/partials/BackLinkViewSpec.scala]: Done

**Pull request merged**